src/App.test.jsx

```diff
@@ -6,6 +6,7 @@ import { useSelector, useDispatch } from 'react-redux';
 
 import App from './App';
 
+jest.mock('./services/api');
 jest.mock('react-redux');
```

Comment:
```suggestion
jest.mock('react-redux');
jest.mock('./services/api');
```

실제 프로젝트 구조상 __mocks__가 src/services 보단 상위 계층이기 때문에

순서를 바꾸는 게 좋겠습니다.

Reply:
done